scrapers/EastBoys.yml

```diff
@@ -0,0 +1,39 @@
+name: "EastBoys"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - eastboys.com/en/tour/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="row cover_detail mt-3 mt-sm-5"]/h2[@class="latest"]/text()
+      Date:
+        selector: //div[@class="update_date text-light"]/text()
+        postProcess:
+          - replace:
+              - regex: '(Added:\s)(\d\d\.\d\d\.\d{4})'
+                with: $2
+              - regex: '(\d\d)(\.)(\d\d)(\.)(\d{4})'
+                with: "$1 $3 $5"
+              - regex: '(\d\d)\s(\d\d)\s(\d{4})'
+                with: "$2 $1, $3"
+          - parseDate: 01 02, 2006
+      Details:
+        selector: //p[@class="description pr-0 pr-lg-3 text-light"]/text()
+#      Performers:
+#        Name:
+#          selector: //html/body/div[2]/div/div[1]/div/span[2]/b/a
+#      URL: //link[@rel="canonical"][1]/@href

```

Comment:
This seems to work for performers as well as adds in tags.
```Performers:
        Name:
          selector: //span[@class="content"]//a
          postProcess:
          - replace:
              - regex: ','
                with: ""
      Tags:
        Name: //div[@class="categories_detail text-light"]/following-sibling::a

Reply:
That does seem to work as well, I went ahead and added/fixed that.

Reply:
:-)